tests/v2/validation/provisioning/hosted/hosted_provisioning_test.go

```diff
@@ -149,6 +152,14 @@ func (h *HostedClusterProvisioningTestSuite) TestProvisioningHostedAKSCluster()
 			clusterToken, err := clusters.CheckServiceAccountTokenSecret(client, clusterName)
 			require.NoError(h.T(), err)
 			assert.NotEmpty(h.T(), clusterToken)
+
+			updatedCluster, err := nodepools.ScalingAKSNodePools(client, clusterResp, clusterName, cloudCredential.ID)
+			require.NoError(h.T(), err)
+			assert.Equal(h.T(), clusterName, updatedCluster.Name)
+
+			importedCluster, err := importing.ImportAKSCluster(client, cloudCredential, importedClusterName)
+			require.NoError(h.T(), err)
+			assert.Equal(h.T(), importedClusterName, importedCluster.Name)
```

Comment:
I still think the same with the previous review and design talk. I think this would be better as another test like we have for custom and node provider provisioning RKE1/RKE2/K3s tests. In my opinion, running this test should only cover provisioning AKS. Currently, it covers imported + provisioning + scaling. I don't have a strong opinion on the scaling, but imported and provisioning tests would be better if separated.

Reply:
Yeah, I agree. As mentioned in an earlier comment, I had forgotten to refactor that part when I pushed the changes yesterday. I'll fix that in a separate commit as well and put EKS and GKE in their own respective folders as well while I'm at it.

Reply:
Temporarily removing the imported test at this time due to offline conversations with Israel and Brandon. I will come back to that test in a separate PR in the near-future. For now, we will focus on specifically refactoring the hosted cluster test structure and adding scaling node pool support.